File: src/main/java/com/circleescape/server/scoreboard/data/Score.java
package com.circleescape.server.scoreboard.data;

import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;
import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@Entity
@Table(name = "SCORES")
public class Score {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(name = "game")
    private Long gameId;

    @Column(nullable = false)
    private String name;

    @Column(nullable = false)
    private double distance;

    @Column
    private LocalDateTime time;

    public Score(Long gameId, String name, double distance, LocalDateTime time) {
        this.gameId = gameId;
        this.name = name;
        this.distance = distance;
        this.time = time;
    }

}

File: src/main/kotlin/com/circleescape/server/scoreboard/data/Score.kt
package com.circleescape.server.scoreboard.data

import java.time.LocalDateTime
import javax.persistence.*

@Entity
@Table(name = "SCORES")
class Score(
    @Column(name = "game") var gameId: Long,
    @Column(nullable = false) var name: String,
    @Column(nullable = false) var distance: Double,
    @Column var time: LocalDateTime,
    @Id @GeneratedValue(strategy = GenerationType.IDENTITY) var id: Long? = null
)